src/TestSuite/Stub/TestExceptionRenderer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         3.5.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\TestSuite\Stub;
+
+use Exception;
+
+/**
+ * Test Exception Renderer.
+ */
```

Comment:
It might make sense to explain a little why this exists. Something like https://github.com/cakephp/cakephp/pull/10777/files#diff-30ad814a76cb9a0911ed10f377ace431R1056